presto-testing/src/main/java/io/prestosql/testing/AbstractTestJoinQueries.java

```diff
@@ -2279,4 +2282,42 @@ public void testMultiJoinWithEligibleForDynamicFiltering()
                         "AND customer.name >= 'Customer#000001463' ",
                 "VALUES 3");
     }
+
+    @Test
+    public void testCardinalityInsensitiveJoin()
```

Comment:
Could you please add tests:
- with FULL join,
- with GROUP BY build symbols,
- with multiple grouping sets?